source/manual/alerts/low-available-disk-space.html.md

```diff
@@ -102,6 +102,34 @@ Steps to investigate postgres db size:
 1. You can choose one of the dbs by doing: `\c <name-of-db>`
 For example: `\c email-alert-api_production` 
 
+## Low available disk space on /mnt/elasticsearch
+
+Usually this is caused by old indices not being closed.
+
+View the dashboard to see how many indices are active on the cluster:
+
+`ssh -L9200:localhost:9200 rummager-elasticsearch-1.api.staging`
+
+Then visit http://localhost:9200/_plugin/head/
+
+Go to the "Indices" tab to see all of the active indices. There should only be one index of each type.
+
+Most of out indices are aliased. The most cases the index in use is the one that the alias points to. This is usually the most recent index. On the dashboard the alias is represented by a colour label.
+
+![Screenshot of a healthy elasticsearch dashboard](images/elasticsearch/elasticsearch-healthy-dashboard.png)
+
+If old indices haven't been deleted, you'll see duplicates.
+
+![Screenshot of an elasticsearch dashboard with duplicate indices](images/elasticsearch/elasticsearch-dashboard-with-duplicate-indices.png)
+
+The indices apart from the ones in use can be closed and then, if no problems arise, deleted.
+
+To close an index, select `Actions -> Close` and then wait a few minutes to make sure everything is ok. If there is a problem, the index can still be re-opened.
```

Comment:
For seeing if everything is ok you might want to suggest monitoring icinga for alerts.